packages/bitcore-wallet-client/src/lib/errors/spec.ts

```diff
@@ -49,9 +49,15 @@ var errorSpec = [{
 }, {
   name: 'INSUFFICIENT_FUNDS_FOR_FEE',
   message: 'Insufficient funds for fee.'
+}, {
+  name: 'INSUFFICIENT_ETH_FEE',
```

Comment:
Why not reusing https://github.com/bitpay/bitcore/pull/2697/files#diff-e57a24c0f3a56c58bf7457e8e42b712cR50
? Adding the same message, per coin, may not be a good idea.

Reply:
Yes, maybe but not sure. I think the error is slightly different. This message are for ERC20 tokens when you don't have enough fee in your ETH linked wallets.